controllers/discordactions.js

```diff
@@ -121,7 +120,41 @@ const addGroupRoleToMember = async (req, res) => {
     return res.boom.badImplementation(INTERNAL_SERVER_ERROR);
   }
 };
+/**
+ * Patch Update user nickname
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+const changeNicknameOfUsers = async (req, res) => {
+  try {
+    const userData = req.userData;
 
+    const { discordId, username } = userData;
+
+    const dataForDiscord = {
+      userName: username,
+      discordId,
+    };
+
+    const authToken = await jwt.sign({}, config.get("rdsServerlessBot.rdsServerLessPrivateKey"), {
+      algorithm: "RS256",
+      expiresIn: config.get("rdsServerlessBot.ttl"),
+    });
+
+    await fetch(`${DISCORD_BASE_URL}/guild/member`, {
+      method: "PATCH",
+      body: JSON.stringify(dataForDiscord),
+      headers: { "Content-Type": "application/json", Authorization: `Bearer ${authToken}` },
+    }).then((response) => response.json());
```

Comment:
it is generally a good practice to wrap `fetch` calls in a `try...catch` block to handle any errors that may occur during the request.

Reply:
in other APIs similar to this, it is done the way I have written. I followed the same pattern of calling fetch inside API call.